src/customers/customer-account-manage.md

```diff
@@ -153,6 +150,9 @@ These columns are available by changing the [column layout]({% link stores/admin
 |Shipping Address|The address where orders are to be shipped.|
 |VAT Number|The value-added tax number that is associated with the customer address. For [digital goods]({% link tax/eu-place-of-supply.md %}) sold in the EU, the VAT is based on the billing address of the customer. <br/><br/>**_Note:_** This field is not the same as the Tax/VAT Number.|
 |Account Lock|Indicates the status of the account. As a security measure, customer accounts can be [locked]({% link customers/password-options.md %}) after too many login attempts. Values: Locked / Unlocked|
+|Status|The current user status. Options: `Active` / `Inactive`|
```

Comment:
```suggestion
|Status|The current user status. Options: Active / Inactive|
```
No need to wrap values in backticks. Same below.

Reply:
Thanks, done 😊